Bigtable system test failure:

/cc @sduskis, @AVaksman 

Introdued in #5614. See: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/7246

```python
______________ TestInstanceAdminAPI.test_create_instance_defaults ______________

self = <tests.system.TestInstanceAdminAPI testMethod=test_create_instance_defaults>

    def test_create_instance_defaults(self):
        from google.cloud.bigtable import enums
    
        ALT_INSTANCE_ID = 'new-def' + unique_resource_id('-')
        instance = Config.CLIENT.instance(ALT_INSTANCE_ID)
>       operation = instance.create(location_id=LOCATION_ID)

tests/system.py:152: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
google/cloud/bigtable/instance.py:241: in create
    clusters=clusters)
google/cloud/bigtable_admin_v2/gapic/bigtable_instance_admin_client.py:387: in create_instance
    request, retry=retry, timeout=timeout, metadata=metadata)
../.nox/sys-2-7/lib/python2.7/site-packages/google/api_core/gapic_v1/method.py:139: in __call__
    return wrapped_func(*args, **kwargs)
../.nox/sys-2-7/lib/python2.7/site-packages/google/api_core/retry.py:260: in retry_wrapped_func
    on_error=on_error,
../.nox/sys-2-7/lib/python2.7/site-packages/google/api_core/retry.py:177: in retry_target
    return target()
../.nox/sys-2-7/lib/python2.7/site-packages/google/api_core/timeout.py:206: in func_with_timeout
    return func(*args, **kwargs)
../.nox/sys-2-7/lib/python2.7/site-packages/google/api_core/grpc_helpers.py:56: in error_remapped_callable
    six.raise_from(exceptions.from_grpc_error(exc), exc)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

value = InvalidArgument("Error in field 'clusters' : Cluster Id : Invalid id for colle...gth should be between [6,30], but found 31 'new-def-7246-1531852904-cluster'",)
from_value = <_Rendezvous of RPC that terminated with:
	status = StatusCode.INVALID_ARGUMEN...een [6,30], but found 31 'new-def-7246-1531852904-cluster'","grpc_status":3}"
>

    def raise_from(value, from_value):
>       raise value
E       InvalidArgument: 400 Error in field 'clusters' : Cluster Id : Invalid id for collection clusters : Length should be between [6,30], but found 31 'new-def-7246-1531852904-cluster'
```

--- a/bigtable/tests/system.py
+++ b/bigtable/tests/system.py
@@ -147,7 +147,7 @@ class TestInstanceAdminAPI(unittest.TestCase):
     def test_create_instance_defaults(self):
         from google.cloud.bigtable import enums
 
-        ALT_INSTANCE_ID = 'new-def' + unique_resource_id('-')
+        ALT_INSTANCE_ID = 'ndef' + unique_resource_id('-')
         instance = Config.CLIENT.instance(ALT_INSTANCE_ID)
         operation = instance.create(location_id=LOCATION_ID)
         # Make sure this instance gets deleted after the test case.

